src/devcenter/azext_devcenter/manual/_help.py

```diff
@@ -210,6 +210,49 @@
                --project-name "{projectName}" --user-id "me"
 """
 
+helps['devcenter dev dev-box list-upcoming-action'] = """
+    type: command
+    short-summary: "Lists upcoming actions on a Dev Box."
+    examples:
+      - name: DevBoxes_ListUpcomingActions
+        text: |-
+               az devcenter dev dev-box list-upcoming-action --dev-center-name "{devCenterName}" \
+--project-name "{projectName}" --name "myDevBox" --user-id "me"
+"""
+
+helps['devcenter dev dev-box delay-upcoming-action'] = """
+    type: command
+    short-summary: "Delays an Upcoming Action."
+    examples:
+      - name: DevBoxes_DelayUpcomingAction
+        text: |-
+               az devcenter dev dev-box delay-upcoming-action --dev-center-name "{devCenterName}" \
+--project-name "{projectName}" --delay-until "2022-09-30T17:00:00Z" --name "myDevBox" \
+--upcoming-action-id "cae4d1f4-94b8-75f2-406d-5f00ae4c1da7" --user-id "me"
+"""
+
+helps['devcenter dev dev-box show-upcoming-action'] = """
+    type: command
+    short-summary: "Gets an Upcoming Action."
```

Comment:
Nit: Should this be "Shows" instead of "Get"?

Reply:
Right now we use "gets" as the short-summary for all the show commands. This is just the description taken from the spec.  Will add as a to do to talk to PM about this if we want to update this.